Scripts/developer_scripts/test_merge_of_branch

```diff
@@ -129,7 +129,7 @@ fi
 
 #check no file contains non-utf8 characters
 echo '.. Checking if non utf-8 characters are used...'
-txt_not_utf8=$(for i in `find . -type f`; do out=`file $i`; if echo $out | grep -q "text"; then if ! (echo $out | egrep -q "UTF-8|ASCII|XML|EPS"); then echo $i; fi; fi; done)
+txt_not_utf8=$(file -N `git ls-files --stage` | grep "text" | egrep -v "UTF-8|ASCII|XML|EPS")
```

Comment:
This patch replaces:
- the `find . -type f` by `git ls-files --stage`: in my working directory, I have more than 200k files, and it falls down to 15k files,
- avoid to fork a `file` and a `grep` for each of them.

This patch probably only works on Linux, where the command line is not limited. A better patch should use `git ls-files --stage | xargs file | grep ...`.